ui/src/app/base/components/NotificationGroup/NotificationGroup.tsx

```diff
@@ -27,7 +28,7 @@ type Props = {
 
 const NotificationGroup = ({ notifications, severity }: Props): JSX.Element => {
   const dispatch = useDispatch();
-  const [groupOpen, toggleGroup] = useVisible(false);
+  const [groupOpen, setGroupOpen] = useState(false);
```

Comment:
This was a very simple hook that was only used here, so I opted to remove the `useVisible` hooks.